bgpd/bgp_attr.c

```diff
@@ -2914,6 +2949,7 @@ void bgp_packet_mpattr_prefix(struct stream *s, afi_t afi, safi_t safi,
 size_t bgp_packet_mpattr_prefix_size(afi_t afi, safi_t safi, struct prefix *p)
 {
 	int size = PSIZE(p->prefixlen);
+
```

Comment:
Whitespace change?

Reply:
Needed for SA: white space after declaration.